README.md

```diff
@@ -5,6 +5,8 @@
 Ruby on Rails application powering
 [https://software.opensuse.org](https://software.opensuse.org)
 
+We hope you'll get involved! Read our [Contributors' Guide](https://github.com/openSUSE/software-o-o/blob/master/CONTRIBUTING.md) for details.
```

Comment:
I think this should be written in present:

`We hope you get involved!`

Reply:
Native speaker disagrees:
>duncanmv: I would say "We hope you'll get involved", it sounds slightly more friendly
> duncanmv: "We hope you get involved" seems more immediate

Reply:
ok... :speak_no_evil: